### PR TITLE
Pyament with stripe implementation

### PR DESCRIPTION
**Pull request by Ishimwe7**
### What does this PR do?
This PR implements the integration of Stripe for handling payments on the front-end of the application.
### Description of Task to be completed?

- Set up Stripe integration with the front-end.
- Create a payment form using Stripe Elements.
- Handle payment processing and validation.
- Implement error handling for payment failures.
- Display success messages and handle post-payment actions.
- Ensure secure handling of payment information.

### How should this be manually tested?

- Start the front-end application using yarn run dev.
-Navigate to the checkout page and initialize payment.
-Fill in the payment form with test card details provided by Stripe (e.g., 4242 4242 4242 4242 for a successful transaction).
-Submit the form and verify that the payment is processed successfully.
-Verify that the success message is displayed and any post-payment actions are executed.

### What are the relevant pivotal tracker stories

https://www.pivotaltracker.com/story/show/187570840

### CODE REVIEW

**Review by bahati10 — APPROVED** [review]
Good job on This Pacifique. Payment is indeed well-implemented

File: src/pages/OrderConfirmation.tsx
Comment (derrick-nuby) [inline src/pages/OrderConfirmation.tsx:1]: I would Suggest you put a stripe logo; so that our users can see that we accept stripe

**Review by derrick-nuby — APPROVED** [review]
Changes looks good; And This PR is ready for merge

**Review by Balinda21 — APPROVED** [review]
The code works well, and this PR is ready for merging.

**Review by kirengamartial — APPROVED** [review]
Good job @Ishimwe7 the payment is working really well

**Review by robsdagreat — APPROVED** [review]

**Review by rodriguecyber — APPROVED** [review]

**Pull request merged**